easybuild/easyblocks/c/cuda.py

```diff
@@ -128,12 +128,14 @@ def sanity_check_step(self):
         extra_files = []
         if LooseVersion(self.version) < LooseVersion('7'):
             extra_files.append('open64/bin/nvopencc')
+        if LooseVersion(self.version) >= LooseVersion('7'):
+            extra_files.append("extras/CUPTI/lib64/libcupti.%s" % shlib_ext)
 
         custom_paths = {
             'files': ["bin/%s" % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] + extra_files +
                      ["%s/lib%s.%s" % (x, y, shlib_ext) for x in chk_libdir for y in ["cublas", "cudart", "cufft",
                                                                                       "curand", "cusparse"]],
-            'dirs': ["include"],
+            'dirs': ["include", "extras/CUPTI/include"],
```

Comment:
This should also be dependent on the CUDA version, so I'd propose an approach like:

```
custom_paths = {
    'files': ...,
    'dirs': ...,
}

if LooseVersion(self.version) < LooseVersion('7'):
    custom_paths['files'].append('open64/bin/nvopencc')
if LooseVersion(self.version) >= LooseVersion('7'):
    custom_paths['files'].append("extras/CUPTI/lib64/libcupti.%s" % shlib_ext)
    custom_paths['dirs'].append('extras/CUPTI/include')